mcp-servers/ansible/deployment.yaml

```diff
@@ -0,0 +1,34 @@
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: ansible-mcp-server
+spec:
+  selector:
+    matchLabels:
+      app: ansible-mcp-server
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: ansible-mcp-server
+    spec:
+      containers:
+      - name: ansible-mcp-server
+        image: quay.io/supathak/ansible-mcp:amd
+        ports:
+          - containerPort: 8000
+            protocol: TCP
+          - containerPort: 8080
+            protocol: TCP
+        env:
+          - name: AAP_TOKEN
+            valueFrom:
+              secretKeyRef:
+                name: aap
+                key: token
+          - name: AAP_URL
+            valueFrom:
+              secretKeyRef:
+                name: aap
+                key: url
```

Comment:
Shouldn't there be yaml files for these secrets too? (without the actual tokens of course 😃)